docs/src/guides/wordpress/deploy/configure.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Configure Wordpress for Platform.sh"
+sidebarTitle: "Configure"
+weight: -100
+toc: false
+description: |
+    Review the basics of what makes up a Platform.sh project, including its three principle configuration files and how to define them for Wordpress.
+---
+
+{{< guides/config-desc name="Wordpress" >}}
+
+## Requests configuration: `routes.yaml`
+
+{{< guides/config-routes template="wordpress-composer" name="Wordpress" >}}
+
+## Service configuration: `services.yaml`
+
+The `services.yaml` file lists the pre-packaged services you need for your application to run. You pick the major version of the service, and Platform.sh updates the patch version periodically so that you always get the newest version when you deploy.
+
+We recommend the latest [MariaDB](/configuration/services/mysql.md) version for Wordpress, although you can also use Oracle MySQL or [PostgreSQL](/configuration/services/postgresql.md) if you prefer. You can add [other services](/configuration/services/_index.md) if desired, such as [Solr](/configuration/services/solr.md) or [Elasticsearch](/configuration/services/elasticsearch.md). You will need to configure Wordpress to use those services as well once the service is enabled.
```

Comment:
Postgresql isn't officially supported, just the MySQL family.  It's probably better to stick to that.